src/Cake.NuGet/Cake.NuGet.csproj

```diff
@@ -26,5 +26,16 @@
     <PackageReference Include="NuGet.Resolver" Version="5.0.0" />
     <PackageReference Include="NuGet.Common" Version="5.0.0" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="2.0.0" />
   </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="Microsoft.NETCore.Platforms" Version="2.2.2" GeneratePathProperty="true">
+      <PrivateAssets>All</PrivateAssets>
+    </PackageReference>
+    <EmbeddedResource  Include="$(PkgMicrosoft_NETCore_Platforms)\runtime.json">
```

Comment:
I'm embedding [this](https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.NETCore.Platforms/runtime.json) in order to build the runtime graph.